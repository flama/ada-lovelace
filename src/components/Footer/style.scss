@import '../../config';

#footer {
  width: 100%;

  > .content {
    @include wrap-mobile;
    padding-top: $space-m;
    padding-bottom: $space-m;

    @include medium-up {
      @include grid(2);
      @include offsetLeft(1);
      padding-top: $space-xs;
      padding-bottom: $space-xs;
    }

    > .link {
      font-size: $font-size-centi;
      text-transform: uppercase;
      letter-spacing: 1px;
      color: $light-purple;
      transition: all 0.3s ease;

      &:hover { color: #fff; }
    }
  }
}
