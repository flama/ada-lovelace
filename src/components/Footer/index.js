import React, { Component } from 'react'

import './style.scss'

class Footer extends Component {

  render()
  {
    return (
      <div id="footer">
        <div className="content">
          <a href="http://flama.is" target="_blank" className="link">
            made with â™¥ by flama
          </a>
        </div>
      </div>
    )
  }
}

export default Footer;
