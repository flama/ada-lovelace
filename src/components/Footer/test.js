import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Footer from '../Footer'

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
