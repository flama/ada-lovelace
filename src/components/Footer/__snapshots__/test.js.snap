// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  id="footer"
>
  <div
    className="content"
  >
    <a
      className="link"
      href="http://flama.is"
      target="_blank"
    >
      made with â™¥ by flama
    </a>
  </div>
</div>
`;
